native/dll/DShowCapture-2/GraphManager.cpp

```diff
@@ -354,17 +362,19 @@ JNIEXPORT void JNICALL Java_sage_DShowCaptureDevice_teardownGraph0
 
 	GETENCODERMUTEX;
 	
-	if ( pCapInfo->captureNum > 0 )
-	{
-		if ( pCapInfo->captures[0]->dwBDAType != pCapInfo->dwBDAType )
-		{
-			TeardownBDAGraph( env, pCapInfo->captures[0] );
-		} else
-		{
-			TeardownBDAGraph( env, pCapInfo->captures[1] );
-		}
-
-	}
+	DETECTED_CAPTURE_NUM = pCapInfo->captureNum;

```

Comment:
Change this line to be:

int detectedCaptureNum = pCapInfo->captureNum;

and then rename below usages of DETECTED_CAPTURE_NUM to be detectedCaptureNum.

Reply:
Done